chapters/en/module2.md

```diff
@@ -0,0 +1,789 @@
+
+</opt>
+
+
+<opt text="False">
+
+Does sorting a bar chart make it easier to read?
+
+</opt>
+
+</choice>
+
+</exercise>
+
+
+<exercise id="10" title='Testing Your Bar Knowledge'>
+
+**Question 1**      
+Why do we avoid bar charts when showing summary statistics such as the mean? 
+
+<choice id="1" >
+<opt text="They don't look the best aestetically." >
+
+Bar plots can be quite useful and beautiful depending on what is communicated!
+
+</opt>
+
+<opt text="Summary statistics are communicated best with line or scatter plots." >
+
+Is a plot communicating just the summary statistics even a good idea?
+
+</opt>
+
+<opt text="They tend to take too long.">
+
+Sometimes this can be an issue with coding but not when it comes to visualization.
+
+</opt>
+
+
+<opt text="They remove a lot of information such as variation amoung the data." correct="true">
+
+Super! 
+
+</opt>
+
+</choice>
+
+
+**Question 2**      
+
+If we wanted to increase the number of bars in the histogram, what would be needed instead of `"???"` in the code below? 
+
+```python
+alt.Chart(data_source).mark_bar().encode(
+    x=alt.X('students', bin=???),
+    y='count()').properties(title='The distribution of students per classroom')
+```
+
+<choice id="2" >
+<opt text="<code>50</code>">
+
+You need to use a helper function to increase the number of bars and not just set `bin` equal to a number. 
+
+</opt>
+
+<opt text="<code>alt.Bin(50)</code>">
+
+There needs to be some sort of argument set within `alt.Bin()`. 
+
+</opt>
+
+
+<opt text="<code>bin=alt.Bin(bins=30)</code>">
+
+You are close but the argument `bins` within the helper function `alt.Bin()` isn't quite right. 
+
+</opt>
+
+<opt text="<code>bin=alt.Bin(maxbins=30)</code>">
+
+Nice!
+
+</opt>
+
+
+</choice>
+
+</exercise>
+
+
+<exercise id="11" title="Practicing Bar Plots with Penguins">
+
+
+**Instructions:**    
+Be patient when running a coding exercise for the first time, it can take a few minutes. 
+
+**When you see `____` in a coding exercise, replace it with what you assume to be the correct code. Run the code to see if you obtain the desired output
+and submit it to validate if you were correct.**
+
+
+Alright, let's bring in a new dataset containing information on penguins. This data is available from [Kaggle](https://www.kaggle.com/parulpandey/palmer-archipelago-antarctica-penguin-data) and originally available from [Allison Horst's GitHub Repository](https://github.com/allisonhorst/palmerpenguins). 
+
+
+
+<codeblock id="penguins">
+
+</codeblock>
+
+
+We are interested in obtaining the number of penguins we collected of each species. 
+
+Tasks: 
+
+- Using a bar plot, visualize the count of penguins for each species in the data. 
+- Choose the appropriate rotation, given the axis labels. 
+- Sort your bars in an appropriate order. 
+- Make sure to give it a title and set the dimensions to a height of 350 and a width of 500.
```

Comment:
The height is too high here and violates the ink-space rule of thumb, please drop down to 100? 150?